Heroes.ReplayParser/MPQFiles/ReplayInitData.cs

```diff
@@ -91,40 +91,21 @@ public static void Parse(Replay replay, byte[] buffer)
 
                 // m_ammId
                 if (replay.ReplayBuild >= 43905 && reader.ReadBoolean())
-                    switch (reader.ReadInt32())
+                {
+                    replay.GameMode = (reader.ReadInt32()) switch
```

Comment:
It looks like the build server failed, and I think it's because this kind of switch statement is a newer C# feature

Do you know what version of .NET Core it became available?  You could try modifying https://github.com/barrett777/Heroes.ReplayParser/blob/master/.travis.yml and see if the build server will pass if you change the .NET version

Or if you don't feel strongly, it would probably be easy to change back to the old style switch statement